src/AppBundle/Controller/Admin/Client/ReportController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace AppBundle\Controller\Admin\Client;
+
+use AppBundle\Controller\AbstractController;
+use AppBundle\Exception\DisplayableException;
+use AppBundle\Form\Admin\UnsubmitReportType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @Route("/admin/report/{id}/", requirements={"id":"\d+"})
+ */
+class ReportController extends AbstractController
+{
+    /**
+     * @Route("manage", name="admin_report_manage")
+     *
+     * @param Request $request
+     * @param $id
+     *
+     * @Template()
+     *
+     * @return array
+     */
+    public function manageAction(Request $request, $id)
+    {
+        $report = $this->getReport($id, []);
+        if (!$report->getSubmitted()) {
+            throw new DisplayableException('Cannot manage active report');
+        }
+
+        $form = $this->createForm(UnsubmitReportType::class, $report);
+        $form->handleRequest($request);
+
+        // edit client form
+        if ($form->isValid()) {
+            $report
+                ->setSubmitted(false)
+                ->setUnSubmitDate(new \DateTime())
```

Comment:
will we need to set the submit flag to false?

Reply:
Since its a PUT to a dedicated endpoint, we dont need to do this here. Can be done at the backend?